L1Trigger/L1CaloTrigger/python/L1CaloJetHTTProducer_cfi.py

```diff
@@ -1,9 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
-L1CaloJetHTTProducer = cms.EDProducer("L1CaloJetHTTProducer",
+l1tCaloJetHTTProducer = cms.EDProducer("L1CaloJetHTTProducer",
     EtaMax = cms.double(2.4),
     PtMin = cms.double(30.0),
-    BXVCaloJetsInputTag = cms.InputTag("L1CaloJetProducer","L1CaloJetCollectionBXV"),
+    BXVCaloJetsInputTag = cms.InputTag("l1tCaloJetProducer","L1CaloJetCollectionBXV"),
     genJets = cms.InputTag("ak4GenJetsNoNu", "", "HLT"),
```

Comment:
Unrelated, but does `ak4GenJetsNoNu::HLT` actually work here? I didn't expect `HLT` as process label for this collection. Might be better to just use `ak4GenJetsNoNu`.